Docs/Generali/analisi_dei_requisiti/02_use_cases.tex

```diff
@@ -1 +1,397 @@
+            \item Viene eseguita l'interazione opportuna da parte dell'utente per la selezione.
+        \end{itemize}    
+    \item \textbf{Postcondizioni:}
+        \begin{itemize}
+            \item Viene evidenziata la selezione sia della barra del grafico selezionata sia della corrispetiva cella della tabella.
+        \end{itemize}    
+    \item \textbf{Scenario:} 
+        \begin{itemize}
+            \item L'utente interagisce con l'ambiente 3D e seleziona una barra del grafico.
+        \end{itemize}
+\end{itemize}
+\paragraph{UC11.1.1: Enfatizzare selezione nella tabella}
+\begin{itemize}    
+    \item \textbf{Attore:} utente;
+    \item \textbf{Descrizione:} Viene evidenziata in maniera opportuna nella tabella la selezione compiuta dall'utente.
+    \item \textbf{Precondizioni:}    
+        \begin{itemize}
+            \item Il caricamento del dataset è avvenuto con successo;
+            \item La generazione della tabella non ha riscontrato errori;
+            \item Viene eseguita l'interazione opportuna da parte dell'utente per la selezione.
+        \end{itemize}    
+    \item \textbf{Postcondizioni:}
+        \begin{itemize}
+            \item Viene evidenziata la selezione nella tabella.
+        \end{itemize}    
+    \item \textbf{Scenario:} 
+        \begin{itemize}
+            \item L'utente interagisce con l'interfaccia e compie un'azione di selezione che viene evidenziata nella tabella.
+        \end{itemize}
+\end{itemize}
+
+\pagebreak
+
+\subsubsection{UC11.2: Selezione di una cella della tabella}
+\begin{figure}[h!]\centering
+    \includegraphics[scale=0.7]{template/images/UC11.2.png}
+    \caption{UC11.2: Selezione di una cella della tabella}
+\end{figure}
+\begin{itemize}    
+    \item \textbf{Attore:} utente;
+    \item \textbf{Descrizione:} L'utente, tramite interazione opportuna, può selezionare una cella all'interno della tabella.
+    \item \textbf{Precondizioni:}    
+        \begin{itemize}
+            \item Il caricamento del dataset è avvenuto con successo;
+            \item La generazione della tabella non ha riscontrato errori;
+            \item Viene eseguita l'interazione opportuna da parte dell'utente per la selezione della cella.
+        \end{itemize}    
+    \item \textbf{Postcondizioni:}
+        \begin{itemize}
+            \item Viene evidenziata la selezione sia della cella della tabella selezionata sia della corrispetiva barra del grafico all'interno dell'ambiente 3D.
+        \end{itemize}    
+    \item \textbf{Scenario:} 
+        \begin{itemize}
+            \item L'utente interagisce con la tabella selezionando una cella.
+        \end{itemize}
+\end{itemize}
+\paragraph{UC11.2.1: Enfatizzare selezione nel grafico}
+\begin{itemize}    
+    \item \textbf{Attore:} utente;
+    \item \textbf{Descrizione:} Viene evidenziata in maniera opportuna nel grafico all'interno dell'ambiente 3D la selezione compiuta dall'utente.
```

Comment:
Scriverei:
Viene evidenziata in maniera opportuna nel grafico 3D la selezione compiuta dall'utente nella tabella.